kotlinx.fuzz.api/src/main/kotlin/kotlinx/fuzz/config/EngineConfig.kt

```diff
@@ -0,0 +1,24 @@
+package kotlinx.fuzz.config
+
+sealed interface EngineConfig
+
+interface JazzerConfig : EngineConfig {
```

Comment:
Right now, `JunitEngine` doesn't depend on any concerete `FuzzEngine`. However, here we have `JazzerConfig` in the API. From my point of view, concerete `FuzzEngines` should be configured somehow independently from the `kotlinx.fuzz` in general, since we might want to change them